src/Angor/Client/Pages/Create.razor

```diff
@@ -851,6 +989,32 @@
         {
             showCreateModal = false;
 
+            var accountInfo = storage.GetAccountInfo(network.Name);
+
+            // refresh UTXOs
+            await _WalletOperations.UpdateDataForExistingAddressesAsync(accountInfo);
+            await _WalletOperations.UpdateAccountInfoWithNewAddressesAsync(accountInfo);
+            storage.SetAccountInfo(network.Name, accountInfo);
+
+            var walletWords = await passwordComponent.GetWalletAsync();
+
+            // is this the correct way to get the address @dangershony @davidgershony
+            var changeAddress = accountInfo.ChangeAddressesInfo.FirstOrDefault(addr => !addr.UtxoData.Any())?.Address;
+            if (string.IsNullOrEmpty(changeAddress))
+            {
+                notificationComponent.ShowErrorMessage("No available change address found.");
+                return;
+            }
+
+            // reconstruct the transaction using refreshed UTXO data
+            signedTransaction = _WalletOperations.AddInputsAndSignTransaction(
```

Comment:
now if a project already exists using the projectidentifier before creating or initializing it. if it exists, the project is updated directly in place instead of creating a new one. this ensures that all updates, including the assignment of creationtransactionid, modify the same project entry throughout the workflow.

Reply:
also, does this have the required outputs? 
https://explorer.angor.io/tbtc/explorer/transaction/04273e7eb6be1d9486d76dc210de92b764544e6c597f470a0ca8735612f78925

Reply:
> also, does this have the required outputs?  
Yes this looks correct